libbeat/processors/registered_domain/registered_domain_test.go

```diff
@@ -0,0 +1,74 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package registered_domain
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func TestProcessorRun(t *testing.T) {
+	var testCases = []struct {
+		Error            bool
+		Domain           string
+		RegisteredDomain string
+	}{
+		{false, "www.google.com", "google.com"},
+		{false, "www.google.co.uk", "google.co.uk"},
+		{false, "google.com", "google.com"},
+		{false, "www.ak.local", "ak.local"},
+		{false, ".", "."},
+		{false, "www.navy.mil", "navy.mil"},
```

Comment:
```suggestion
		{false, "www.navy.mil", "navy.mil"},
		{false, "localhost", "localhost"},
```

Suggestion for an additional test for a very common value

Reply:
That's not the actual behavior. Since it's an unqualified hostname there isn't any domain name for it to return.

Reply:
Gotcha 👍